client/.github/workflows/ci-client.yaml

```diff
@@ -0,0 +1,23 @@
+name: CI
+on:
+ pull_request:
+jobs:
+ check:
+   runs-on: ubuntu-22.04
+   steps:
+     - uses: actions/checkout@v4
+     - name: Install Node
+       uses: actions/setup-node@v4
+       with:
+         node-version: 'lts/*'
```

Comment:
do we have a nvm/fnm rc file that will help make sure we are developing on a local node version that matches the one we run in CI?

Reply:
The typical way to enforce this is with `engines` in package.json. However, I think this type of issue should only be investigated if it becomes a problem. Generally, we should try to stay on LTS. It has a very generous window:
https://nodejs.org/en/about/previous-releases